### DISCUSSION TITLE
2024-10-25 - Stretching at your Desk - Chewing the Fat 📈

### DISCUSSION THREAD

**Discussion by bradystroud** [SSWConsulting/SSW.Rules.Content#9480] [Chewing the Fat]
https://www.ssw.com.au/rules/stretch-at-work/

Which stretch felt like it made the most difference?
    # 1 Thoracic Spine
    # 2 Chest Opener
    # 3 Hip Flexor

**Comment by drwharris** [comment]
>1 Which stretch felt like it made the most difference?

I do the Hip Flexor stretch every morning and night already

I also use a foam roller to lay on to help open my chest

**Comment by TobyChurches** [comment]
> Which stretch felt like it made the most difference?

2 Chest Opener

This is a great rule and something that we should be trying to improve. I often forget about my posture - We need `SSWStretchMasters` to call out bad posture when they see it 😅

**Comment by MarcusIrmscherSSW** [comment]
I found the Thoracic Spine stretch made the most difference. 

Its a great idea to draw focus to the benefits of stretching. We all know it helps but it easy to overlook especially when you're busy. 💪

**Comment by ozdemir-mehmet** [comment]
Loved them all.

**Reply by jakebayliss** [reply]
Oh im sure you did

**Comment by jakebayliss** [comment]
Which stretch felt like it made the most difference?
I do lots to help this already when running/doing pull ups :)

**Comment by brookjeynes-ssw** [comment]
Taking small breaks throughout the workday to stretch will not only improve your physical but also mental.

I think it's also really important to look at ways the tools you use in your daily worklife can help improve your wellbeing / posture, i.e. using chairs that sit upright, using split keyboards to keep your wrists from bending inwards.

**Comment by calumjs** [comment]
Stretching is a good start, I'd recommend going to the gym as well

**Comment by matt-goldman-ssw** [comment]
Great rule and awesome video. Personally if I don't stretch regularly I get all kinds of problems, specifically in my hips and ITB.

**Comment by joshbermanssw** [comment]
Im a big fan of the thoracic spine stretch - usually perform these quite often throughout the day

**Comment by sam-s-maher** [comment]
Thoracic Spine, it's always the back for me. Quality video Jeoffrey!

**Comment by RobJT-wq** [comment]
I felt the hip flexor stretch the most.

**Comment by leahy268** [comment]
Thoracic Spine - Always the back for me.  Need to probably actually leave my desk during the day sometimes.

**Comment by ricksu978** [comment]
> Which stretch felt like it made the most difference?

\# 2 Chest Opener

**Comment by steven0x51** [comment]
> Which stretch felt like it made the most difference?

2 Chest Opener

**Comment by Tanya-Leahy** [comment]
>Which stretch felt like it made the most difference?

2 Chest Opener

**Comment by StefStar** [comment]
Thoracic Spine - Best for me as it relieves and stretches my back in a great way and is almost muscle memory for me to do each hour.

**Comment by tiagov8** [comment]
# 1 Thoracic Spine - My posture isn't great, and my back pays the highest price.

**Comment by miichaelsmedley** [comment]
> https://www.ssw.com.au/rules/stretch-at-work/
> 
> Which stretch felt like it made the most difference? # 1 Thoracic Spine # 2 Chest Opener # 3 Hip Flexor

1 Thoracic Spine - Really opens up the body.

**Comment by ncn-ssw** [comment]
> Which stretch felt like it made the most difference?

1 - Thoracic Spine

**Comment by Lewkans** [comment]
> Which stretch felt like it made the most difference?

2 - Chest Opener

**Comment by chrisschultzssw** [comment]
> Which stretch felt like it made the most difference?

1 Thoracic Spine

**Comment by vladislav-kir** [comment]
> Which stretch felt like it made the most difference?

2 Chest Opener

**Comment by tkapa** [comment]
Hip flexor stretches.

I already suffer from chronic low back pain, so I've needed to start stretching regularly and also before and after my workouts.

**Comment by jimmy1992abc** [comment]
I just noticed that I am doing Chest Opener every day because it feels comfy.
I should also try the others, especially Hip Flexor.

**Comment by BenDunkerleySSW** [comment]
Thoracic Spine - I have had spine problems for years, and this one is always super relieving.

**Comment by edkranz** [comment]
> Which stretch felt like it made the most difference?

Chest Opener — I find I do this naturally, however enforcing a 30 second minimum stretch is important!
Also dynamic stretches tend to be more effective than static stretches for me personally.

**Comment by awaltos** [comment]
They were all great. Probably the chest one felt the best.

Healthy body, healthy mind

**Comment by danielmackay** [comment]
> Which stretch felt like it made the most difference?

They were all pretty good.  Thoracic Spine, followed by Hip Flexor.

Great video @jeoffreyfischer. This has made me more aware of moving regularly during the day, and has encouraged me to stand more at my desk.

**Comment by jeoffreyfischer** [comment]
> Which stretch felt like it made the most difference?

Thoracic Spine always makes me crack a bit - I have a tight upper back 🫠
Hip Flexor also feels like a relief because my hips lack mobility

**Comment by bettybondoc** [comment]
> Which stretch felt like it made the most difference?

Thoracic Spine

Hip Flexor is ok but uncomfortable to do in the office 🫥

**Comment by wicksipedia** [comment]
I take the standing reminders from my watch seriously! I used to have to do supine twists all the time to crack my back but moving around more helped alleviate that issue

![image](https://github.com/user-attachments/assets/e9d4b4fa-8985-4553-b2cd-dddefa6041fe)
bad example: sitting all day makes me feel like this

![](https://media.tenor.com/X_cvrSrqFSkAAAAC/henrycavill-arm.gif)
good example: ready for the next teams call

**Comment by Calinator444** [comment]
> Which stretch felt like it made the most difference?

Thoracic Spine

I've basically been sitting at desks my whole life and I think I have bad posture as a result 🥲

**Comment by jimmy1992abc** [comment]
By the way, I believe there are some other ways to stretch yourself and easily to be set up in our office.
E.g.,
[![Craziest Racing Simulator Perspective Ever](https://i.makeagif.com/media/5-21-2015/3tdaP4.gif)](url)

**Comment by MattParkerDev** [comment]
Awesome video Jeoffrey!

However, the images on the rule don't line up at all with the instructions or how they were demonstrated in the video, and are clearly AI generated.

**Reply by tiagov8** [reply]
+1 real images would be better

**Comment by bahjatalaadelssw** [comment]
> Which stretch felt like it made the most difference?
All the stretches helped to relief tension

**Comment by Ben0189** [comment]
1 Thoracic Spine

**Comment by stephanfakossw** [comment]
Awesome video dude... Kudos on filming devs in their natural habitat!
You're absolutely correct, we should stretch more - I'll add to the routine.

**Comment by Levijj22** [comment]
Awesome video Jeoffrey! Great reminders

**Comment by SamBWagner** [comment]
> Which stretch felt like it made the most difference?
I've solved most of my pain that I was getting in my wrists and hips by exercising and moving to a split keyboard.

The pain I have now is in my upper back/neck and it's likely due to the way I sleep/the fact that I don't have monitors (my fault on the monitor front lol).

Cool video, ergonomics is a super dicey field though.

**Comment by jimmidier** [comment]
> Which stretch felt like it made the most difference?

All great stretches. I like the Chest Opener a bit more. It helps the breath to feel smoother

**Comment by Aibono1225** [comment]
> Which stretch felt like it made the most difference?

Love Thoracic Spine and Chest Opener

**Comment by jerryluossw** [comment]
> Which stretch felt like it made the most difference?

Great video, I like Thoracic Spine the most, and Chest Opener is also great.

**Comment by tino-liu** [comment]
> Which stretch felt like it made the most difference?

Chest Opener & Hip Flexor

**Comment by ozair-ashfaque** [comment]
2 Chest Opener

**Comment by babakamyljanovssw** [comment]
> Which stretch felt like it made the most difference?

Chest Opener

**Comment by amankumarrr** [comment]
Awesome video Jeoffrey, I like the first one "Thoracic Spine", good for the spine and can be done while sitting.

**Comment by PennyWalker** [comment]
1 Thoracic Spine - I really felt how tight the middle of my back is! Great video! You explained it so well that you could tell you're an instructor! I agree with Matt P. about the images - I'd get Jeoffery to demo the photo and add his pics instead :)

**Comment by Marxoz** [comment]
#3 - It is all in the hips

**Comment by JackDevAU** [comment]
#3 - Now to just keep at it 😅
